CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ To install them you would need to run `install` command:
 poetry install
 ```
 
-To activate your `virtualenv` run `poetry shell`.
+To activate your `virtualenv` run `eval $(poetry env activate)`.
```

Comment:
Since poetry 2, `poetry shell` displays a warning:
```
╰─ ➜ poetry shell

Looks like you're trying to use a Poetry command that is not available.

Since Poetry (2.0.0), the shell command is not installed by default. You can use,

  - the new env activate command (recommended); or
  - the shell plugin to install the shell command

Documentation: https://python-poetry.org/docs/managing-environments/#activating-the-environment

Note that the env activate command is not a direct replacement for shell command.
```